Author: Guillem Jover <guillem@hadrons.org>
Description: Disable assembler optimized triangle setup functions for glide2
 They are not PIC ready.
Bug-Debian: #376812
Origin: vendor

---
Index: b/glide2x/cvg/glide/src/makefile.linux
===================================================================
--- a/glide2x/cvg/glide/src/makefile.linux
+++ b/glide2x/cvg/glide/src/makefile.linux
@@ -23,7 +23,7 @@ TAG_REGEXP 	= $(BUILD_ROOT)/$(FX_GLIDE_H
 # Compile for specific hardware
 ifeq ($(FX_GLIDE_HW),cvg)
 FX_GLIDE_REAL_HW=       1
-FX_GLIDE_CTRISETUP =    0
+FX_GLIDE_CTRISETUP =    1
 
 HWSPEC          =       fifo.c
 LCDEFS          +=      -DCVG \
Index: b/glide2x/h3/glide/src/makefile.linux
===================================================================
--- a/glide2x/h3/glide/src/makefile.linux
+++ b/glide2x/h3/glide/src/makefile.linux
@@ -17,6 +17,8 @@
 # 
 # COPYRIGHT 3DFX INTERACTIVE, INC. 1999, ALL RIGHTS RESERVED
 
+FX_GLIDE_CTRISETUP = 1
+
 ifeq ($(FX_CHRIS_DENIS_ANTHONY_HACK),1)
 LCDEFS          +=       -DCHRIS_DENIS_ANTHONY_HACK=1
 endif
@@ -88,9 +90,10 @@ GL_AMD3D        =       1
 ifneq ($(GL_AMD3D),)
 LCDEFS          +=      -DGL_AMD3D
 LIBOBJS         +=      xtexdl_3dnow.o
-CFILES          +=      xtexdl_def.c
 endif
 
+CFILES          +=      xtexdl_def.c
+
 # Do culling test in sw for independent triangles
 CULL_MODE       =
 ifneq ($(FX_GLIDE_HW_CULL),1)
@@ -129,7 +132,7 @@ endif
 ifeq ($(FX_GLIDE_CTRISETUP),1)
 ASMTRISETUP     =
 ATSOPTS         =       -DGLIDE_USE_C_TRISETUP 
-CFILES          =       gxdraw.c
+CFILES          +=       gxdraw.c
 else
 DSPOPTS         +=      -DGLIDE_DISPATCH_SETUP=1 -DGLIDE_PACKED_RGB=0
 
